### ISSUE TITLE
KeepKey device defined as a Trezor device

### ISSUE THREAD

**Issue by doolio** [romanz/trezor-agent#441]
https://github.com/romanz/trezor-agent/blob/3c911e99a0394278104564092225d67c75e74b99/libagent/device/keepkey.py#L20

Should this not be:

````python
class KeepKey(interface.Device):
````

**Comment by romanz** [comment]
I used [inheritance](https://docs.python.org/3/tutorial/classes.html#inheritance) to re-use TREZOR agent implementation with KeepKey class.

**Comment by doolio** [comment]
Right. I guess I was curious to why this was done for the KeepKey and not the other hardware devices. Feel free to close this if you don't wish to change how this is implemented. If you would like to align the KeepKey device with the others for consistency then I could submit a PR for consideration.

**Comment by romanz** [comment]
Keepkey support has been removed in #511.
